EIPS/eip-crosschain-id.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: <to be assigned>
+title: Crosschain Identifier Specification
+author: Weijia Zhang <weijia@wanchain.org>, Peter Robinson <peter.robinson@consensys.net>
+discussions-to:
```

Comment:
Please update with URL to discuss this EIP. `https://ethereum-magicians.org` is preferred.

Reply:
This EIP cannot be merged until this is fulfilled.